### PR TITLE
no need to add it to  array if there is no logFn

### PR DESCRIPTION
**Pull request by clintonyeb**
Will love to receive your feedback and possibly how I can help even if this pull request is rejected. Thanks guys for your hard work, really appreciated!

### CODE REVIEW

File: lib/http-server.js [hunk 1]
```diff
@@ -58,13 +58,13 @@ function HttpServer(options) {
 
   var before = options.before ? options.before.slice() : [];
 
-  before.push(function (req, res) {
-    if (options.logFn) {
+  if (options.logFn) {
+    before.push(function (req, res) {
       options.logFn(req, res);
-    }
+    })
```
Comment (no1melman) [inline lib/http-server.js]: missing semi-colon

File: lib/http-server.js [hunk 2]
```diff
@@ -58,13 +58,13 @@ function HttpServer(options) {
...
 
     res.emit('next');
```
Comment (no1melman) [inline lib/http-server.js]: This needs to be inside the `before.push(function (req, res) { })` otherwise it will be undefined

**Comment by BigBlueHat** [comment]
Huge thanks for the code review @no1melman!

@clintonyeb would be great to have these tweaks made and the build passing. 😃 Thanks for the work!

**Comment by no1melman** [comment]
@BigBlueHat no problem, just thought I would give my two cents!

**Comment by thornjad** [comment]
Since the source branch has been deleted, changes can't be made on this PR, so there's no path forward. @no1melman please feel free to make a new PR if you'd like!

**Comment by thornjad** [comment]
Reimplemented in #586

**Pull request closed without merging**